tests/unit/lib/retry_tests.py

```diff
@@ -117,4 +117,4 @@ def test_budget_overrides_backoff(self, sleep, time):
 
         time.return_value = 1
         with self.assertRaises(StopIteration):
-            next(retries)
+            next(retries)[1]
```

Comment:
I think this is a leftover change from the retry policy change and should be reverted?